### PR TITLE
updated broken document link for out of band swaps

### PR DESCRIPTION
**Pull request by git-ayas**
Simple doc fix. Cheers and keep up the work !

### CODE REVIEW

**Comment by git-ayas** [comment]
Btw sorry but I'm not able to add a label

File: src/ws/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ WebSockets extension support two configuration options:
 The example above establishes a WebSocket to the `/chatroom` end point. Content that is sent down from the websocket
 will
 be parsed as HTML and swapped in by the `id` property, using the same logic
-as [Out of Band Swaps](https://htmx.org/reference/hx-swap-oob.md).
+as [Out of Band Swaps](https://htmx.org/docs/#oob_swaps).
```
Comment (Telroshan) [inline src/ws/README.md:51]: Hey, good catch of a dead link, thanks!

I think we maybe should link to https://htmx.org/attributes/hx-swap-oob/ instead, which is more complete & a page dedicated to OOB swapping

Comment (git-ayas) [reply src/ws/README.md:51]: Yup. Looks like a more complete doc.  Let me update the PR

**Review by Telroshan — APPROVED** [review]
Thanks!

**Pull request merged**